otwcode/otwarchive
# AO3-4873 Implements strong parameters for Tag Set Nominations

# Pull Request Checklist

* [x] Have you added tests for any changed functionality? (N/A)

## Issue

https://otwarchive.atlassian.net/browse/AO3-4873

## Purpose

This PR adds strong parameters to the Tag Set Nominations controller and includes ForbiddenAttributes protection to the relevant TagSetNomination model.

## Testing

1. Log in
2. Browse > Tags > Tag Sets > New Tag Set
3. Fill in Title
4. Check "Currently taking nominations?"
5. Set fandom, character, relationship, and freeform nomination limits to desired numbers
6. Submit
7. Follow "Nominate" link
8. Fill in the nominations form
9. Submit
10. Follow "Edit"
11. Change some things on the form
12. Submit

## References

No other references.

## Credit

Littlelines - David Stump

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

(he/him)
